common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -475,6 +484,96 @@ def test_navigation_buttons(self):
         self.courseware_page.click_previous_button_on_bottom()
         self.assert_navigation_state('Test Section 1', 'Test Subsection 1,1', 2, next_enabled=True, prev_enabled=True)
 
+        sequence_ui_events = self.wait_for_events(event_filter=filter_sequence_ui_event)
+        next_events = [ev for ev in sequence_ui_events if ev['name'] == 'edx.ui.lms.sequence.next_selected']
+        previous_events = [ev for ev in sequence_ui_events if ev['name'] == 'edx.ui.lms.sequence.previous_selected']
+        tab_events = [ev for ev in sequence_ui_events if ev['name'] == 'edx.ui.lms.sequence.tab_selected']
+        legacy_next_events = [ev for ev in next_events if ev['event_type'] == 'seq_next']
+        legacy_previous_events = [ev for ev in previous_events if ev['event_type'] == 'seq_prev']
+        self.assertEqual(len(sequence_ui_events), 7)
+        self.assertEqual(len(next_events), 3)
+        self.assertEqual(len(previous_events), 3)
+        self.assertEqual(len(tab_events), 1)
+        self.assertEqual(len(legacy_next_events), 1)
+        self.assertEqual(len(legacy_previous_events), 1)
```

Comment:
Are these assertions necessary? It seems like this is covered by `assert_events_match`